src/en/about/guide.md

```diff
@@ -56,40 +56,40 @@ The standard Lemmy interface has a single feed. You can change the content of th
+| !\[Image\](http://url/a.png)                                                             | !\[Image\]\[1\]<br>⋮ <br>\[1\]: http://url/b.jpg                               | ![Markdown](https://commonmark.org/help/images/favicon.png)                                          |
+| \> Blockquote                                                                            |                                                                                | <blockquote>Blockquote</blockquote>                                                                  |
+| \* List <br>\* List <br>\* List                                                          | \- List <br>\- List <br>\- List <br>                                           | _ List <br>_ List <br>\* List <br>                                                                   |
+| 1\. One <br>2\. Two <br>3\. Three                                                        | 1) One<br>2) Two<br>3) Three                                                   | 1. One<br>2. Two<br>3. Three                                                                         |
+| Horizontal Rule <br>\---                                                                 | Horizontal Rule<br>\*\*\*                                                      | Horizontal Rule <br><hr>                                                                             |
+| \`Inline code\` with backticks                                                           |                                                                                | `Inline code` with backticks                                                                         |
+| \`\`\`<br>\# code block <br>print '3 backticks or'<br>print 'indent 4 spaces' <br>\`\`\` | ····\# code block<br>····print '3 backticks or'<br>····print 'indent 4 spaces' | \# code block <br>print '3 backticks or'<br>print 'indent 4 spaces'                                  |
+| ::: spoiler hidden or nsfw stuff<br>_a bunch of spoilers here_<br>:::                    |                                                                                | <details><summary> hidden or nsfw stuff </summary><p><em>a bunch of spoilers here</em></p></details> |
+| Some ~subscript~ text                                                                    |                                                                                | Some <sub>subscript</sub> text                                                                       |
+| Some ^superscript^ text                                                                  |                                                                                | Some <sup>superscript</sup> text                                                                     |
```

Comment:
Based on the rich diff it looks like the list section isnt rendered properly anymore.

Reply:
Prettier isn't doing anything against commonmark, so its probably a github rendering issue. If its still a problem I can revert.